app/backplane/power_module/CMakeLists.txt

```diff
@@ -4,11 +4,35 @@
 # Copyright (c) 2023 Launch Initiative
 # SPDX-License-Identifier: Apache-2.0
 
+set(BOARD_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
+
+# Guarantee that prj.conf is the default and first configuration file to be read
+
+list(APPEND CONF_FILE ${CMAKE_CURRENT_SOURCE_DIR}/core.conf)
+
+if (DEBUG)
+    message(STATUS "Adding debug configuration")
+    list(APPEND CONF_FILE ${CMAKE_CURRENT_SOURCE_DIR}/debug.conf)
+endif ()
+
+if (BOARD STREQUAL "native_sim")
```

Comment:
if the sim.conf is only applicable to native_sim, you can put it as boards/native_sim.conf and it will be added in automagically so you don't have to do silly cmake things

Reply:
Thats not as cool though